verify.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+travis_tag=$1
```

Comment:
You are not calling this with an argument, so this will always be an empty string. You probably want to use `$TRAVIS_TAG` in this script or in the command that calls it. Alternatively you can run this script in the deploy step, which only gets run if the same conditional that you have on like 8 is true. See https://github.com/atomist/end-user-documentation/blob/master/.travis.yml#L20